api_docs/changelog.md

```diff
@@ -150,10 +150,10 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 168**
 
-* [`PATCH /realm/user_settings_defaults`](/api/update-realm-user-settings-defaults),
-  [`POST /register`](/api/register-queue),
-  [`PATCH /settings`](/api/update-settings): Replaced the `realm_name_in_notifications`
-  boolean field with an integer field `realm_name_in_email_notifications_policy`.
+* [`POST /register`](/api/register-queue), [`PATCH /settings`](/api/update-settings),
+  [`PATCH /realm/user_settings_defaults`](/api/update-realm-user-settings-defaults):
+  Replaced the user setting `realm_name_in_notifications` boolean with
+  `realm_name_in_email_notifications_policy` integer.
```

Comment:
This phrasing doesn't parse for me. Perhaps this?
```suggestion
  Replaced the boolean user setting `realm_name_in_notifications` with
  an integer `realm_name_in_email_notifications_policy`.
```

I think it's that in both "the user setting `foo`" and "the `foo` boolean", I'd read "user setting" or "boolean" respectively as the head of the phrase — it's a user setting, or a boolean, respectively. And correspondingly in "the user setting `foo` boolean" they both want to be the head of the phrase, and I don't have a way to read it with either of them as a modifier instead.

So "the boolean user setting `foo`" resolves it by letting "boolean" be a modifier: it's a user setting, which is boolean, and which is named `foo`.

Reply:
Yes, that reads much more clearly. Updated as such, thanks!